test/e2e/e2e-ssl/src/docker/certs/server-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Are these cert files alway valid? No expired time?

Reply:
2 years. Is it enough?

Reply:
My point is, this is for test. Could we set it forever? Or let's say 100 years?

Reply:
Now we get the  100 years/ century certificates.